.github/workflows/macos-build.yml

```diff
@@ -29,6 +29,8 @@ jobs:
       - name: Configure build environment
         run: |
           brew install llvm
+          brew install ninja
```

Comment:
Use a single brew install to install both.

Reply:
https://github.com/rime/librime/pull/718/commits/2c918b0a0c7e6f4bb18a1f21c8d8e4577e49f97c